pages/api/word-list/[user].js

```diff
@@ -3,35 +3,30 @@ import Entry from "@/db/models/Entry";
 
 export default async function handler(request, response) {
   const { user } = await request.query;
+
   await dbConnect();
 
-  // Should I put something in case the user has not been loaded yet?
-  // It always takes the session a bit to load in the active user.
-  // I got a lot of error messages when I left the value as just undefined. So I changed it to "loading"
-  // Something like the following?
-  // if (user === "loading") {
-  //   response.status(102).json({ status: "processing" });
-  // }
+  if (user === "loading" || user === "default") {
+    return;
+  }
```

Comment:
Nice early return. Unfortunatly your backend doesn't communicate to your frontend that something went wrong and you will get a timeout. you should send a response before terminating the handler function.